app/blog/templates/blog/parts/chatbox.html

```diff
@@ -7,27 +7,44 @@
+                    <p class="chatbox__heading--header">Ezra</p>
+                    <p class="chatbox__time--header"> {% now "jS F Y" %} </p>
                 </div>
+                <div class="chatbox__close--header">
+                    <button aria-label="Close chatbox">
+                        <img src="{% static 'svg/close-icon.svg' %}" alt="Close icon">
+                    </button>
+                </div>
+            </div>
+            <div class="chatbox__messages">
+                <div class="messages__item messages__item--bot"> I can answer questions about any blog post on this
+                    site! </div>
+                <div class="messages__item messages__item--bot"> Hi, I'm Ezra! </div>
             </div>
-            <div class="chatbox__messages"></div>
             <div class="chatbox__footer">
                 <form id="chatbox-submit">
                     {% csrf_token %}
-                    <textarea name="question-text-area" id="question-text-area" placeholder="Write a message..."
-                        hx-trigger="keyup[key === 'Enter' && !shiftKey]" hx-post="/answer-with-gpt/"
-                        hx-include="#question-text-area" hx-target=".chatbox__messages" hx-swap="afterbegin"></textarea>
-                    <button class="chatbox__send--footer send__button" hx-post="/answer-with-gpt/"
-                        hx-include="#question-text-area" hx-target=".chatbox__messages" hx-swap="afterbegin"
-                        aria-label="Send message">Send</button>
+                    <div class="chatbox__input-container">
+                        <textarea name="question-text-area" id="question-text-area" placeholder="Send a message..."
+                            hx-trigger="keyup[key === 'Enter' && !shiftKey]" hx-post="/answer-with-gpt/"
+                            hx-include="#question-text-area" hx-target=".chatbox__messages"
+                            hx-swap="afterbegin"></textarea>
+                        <button class="chatbox__send--footer send__button" hx-post="/answer-with-gpt/"
+                            hx-include="#question-text-area" hx-target=".chatbox__messages" hx-swap="afterbegin"
+                            aria-label="Send message">
+                            <img src="{% static 'svg/send-icon.svg' %}" alt="Send icon">
+                        </button>
+                    </div>
                 </form>
             </div>
+            <div class="chatbox__footer--small">
+                <p>Powered by <a href="https://blogthedata.com">BlogTheData</a></p>
+            </div>
         </div>
         <div class="chatbox__button">
-            <button aria-label="Toggle chatbox"><img id="chatbox-icon" src="{% static 'svg/chatbox-icon.svg' %}"
-                    alt="Chatbox icon" /></button>
+            <button aria-label="Toggle chatbox">
+                <img id="chatbox-icon" src="{% static 'svg/chatbox-icon.svg' %}" alt="Chatbox icon" />
```

Comment:
If you end up changing the icon color on hover, you might need to embed the svg directly in the HTML instead of using the `img` tag.